pkg/blob/controllerserver.go

```diff
@@ -752,6 +757,17 @@ func (d *Driver) copyBlobContainer(_ context.Context, req *csi.CreateVolumeReque
 				return err
 			case util.AzcopyJobNotFound:
 				klog.V(2).Infof("copy blob container %s to %s", srcContainerName, dstContainerName)
+				if storageIdentityClientID != "" {
+					klog.V(2).Infof("use msi client id to authorize azcopy")
+					_, err = exec.Command("export", "AZCOPY_AUTO_LOGIN_TYPE=MSI").CombinedOutput()
```

Comment:
use golang coding style, and also print out error msg in out	
```
if out, err = exec.Command("export", "AZCOPY_AUTO_LOGIN_TYPE=MSI").CombinedOutput(); err != nil {
   // pls also print out error msg in out	
   return err
}
```

Reply:
done, use os.Setenv due to using export command has some issue